frontend/components/Editor.js

```diff
@@ -378,7 +381,7 @@ export class Editor extends Component {
                 this.setState(
                     immer((state) => {
                         for (let cell of cells_to_add) {
-                            state.cell_inputs_local[cell.cell_id] = cell
+                            state.notebook.cell_inputs[cell.cell_id] = cell
```

Comment:
`update_notebook()`?

Reply:
this is actually done a bit below, so I'm just removing this completely